tools/make_class_power.py

```diff
@@ -108,6 +109,11 @@ def _build_cosmology_params(config):
         gparams['tol_ncdm_synchronous'] = 1e-5
         gparams['tol_ncdm_bg'] = 1e-10
         gparams['l_max_ncdm'] = 50
+        #This disables the fluid approximations, which make P_nu not match camb on small scales.
+        #We need accurate P_nu to initialise our neutrino code.
```

Comment:
You mentioned fluid approximation is faster? Why is it 'needed' to have accurate P_nu at small scales for the initialization?